run-tests.sh

```diff
@@ -10,7 +10,7 @@
 export PYTEST_ADDOPTS='docs tests invenio_app'
 
 pydocstyle invenio_app tests docs && \
-isort -rc -c -df && \
+isort invenio_app/*.py tests/*.py --check-only --diff && \
```

Comment:
Can you verify that also subdirectories inside invenio-app is being checked. I know there are not anyone know, but there might be in the future, so better to ensure the wildcard matching works for everything. You can test by simply adding a directory and a python file with a bad import.....

Reply:
I just did this. I created a new command `isort invenio_app/**/*.py  tests/**/*.py --check-only --diff` that does a 1 level subdirectory check. However if there are no python files inside of the subdierctory then it returns a warning `WARNING: Unable to parse file tests/**/*.py due to [Errno 2] No such file or directory: '/Users/rodrigo/Desktop/invenio-app/tests/**/*.py'`.
How should I proceed?

Reply:
Just did it! :)